src/main/java/ui/UI.java

```diff
@@ -143,6 +143,14 @@ private void login(boolean isBypassLogin) {
         getMainWindowHandle(mainStage.getTitle());
     }
 
+    private void launchSampleBoard(){
+        BoardAutoCreator boardCreator = new BoardAutoCreator(this, panels, prefs);
+        if (prefs.getAllBoardNames().contains(boardCreator.SAMPLE_BOARD)){
+            prefs.removeBoard(boardCreator.SAMPLE_BOARD);
```

Comment:
I was thinking maybe if we wanted to update the sample board in the future the user could get a new version of the sample board. But now that I think, the user himself would then not be a first time user and he certainly would not delete repo files from the store folder himself. The `Auto-create` itself would enable the creation of a new sample board if need be! I guess this if statement is not necessary. I'll get rid of it.